HazelcastCloudTests/javahazelcastcloudtests/src/test/java/HazelcastCloudManager.java

```diff
@@ -16,22 +16,25 @@
 import java.util.UUID;
 import java.util.concurrent.TimeUnit;
 
-public class HazelcastCloudManager {
+public class CloudManager {
```

Comment:
So, we duplicate the code in the remote controller here? Can't we re-use it?

Reply:
🤔 How? Do you propose to download it?

Reply:
No, I meant adding a dependency to the remote controller in pom.xml